packages/orchestration/src/exos/cosmos-interchain-service.js

```diff
@@ -203,7 +186,7 @@ const prepareCosmosOrchestrationServiceKit = (
             controllerConnectionId,
             opts,
           );
-          const portAllocator = getPower(this.state.powers, 'portAllocator');
+          const { portAllocator } = this.state;
```

Comment:
yay for simpler. what happens if we need to provide new powers?

Reply:
I'm not entirely certain and figured it'd be easiest to get feedback in the PR.

I based `reserved` on what we did for `localchain.js`. This would indicate we can only add more powers here. 

A more recent read of #7337 leads me to believe StateShape only matters if we specify `opts.stateShape` for the 5th arg?  We can remove `reserved` and rely on `...powers` in the initState function for new powers? If not, it'd be great to understand better.

Reply:
> only matters if we specify opts.stateShape

no; you can't add new properties, even without stateShape
> Once the state is defined by the init function (3rd arg), properties cannot be added or removed.
 -- https://docs.agoric.com/guides/zoe/contract-upgrade.html

`reserved` lets us put a new record there, or a store like `powers` was, should we need one.

Also, lacking that, there's the "side table" approach used to add non-vbank asset support to the smart wallet in #8071.

Reply:
Thanks for clarifying @dckc . I added [9e16702](https://github.com/Agoric/agoric-sdk/pull/9810/commits/9e1670228e94a9921e66dd429affc83c60850053) which spreads `powers` after `reserved` and defines `reserved` on `OrchestrationPowers`. This way, we can define extra powers via `powers.reserved`  if needed.